index.html

```diff
@@ -176,7 +173,7 @@ <h3 class="project-title">Multi-Post Stories</h3>
         </div>
       </section>
       
-      <section class="rose-full-details">
+      <section class="rose-full-details" id="aboutme">
         <div class="about-rose">
           <div class="about-title-section">
             <h1 class="about-title padding">About Myself</h1>
```

Comment:
- [ ] You did an excellent job organizing the heading hierarchy on your HTML page :+1:. However, it's important to note that it's required to use only one `h1` element per HTML page. This is because the `h1` element is meant to be the main heading and should represent the primary topic of the page. Using more than one `h1` element can make it difficult for search engines and assistive technologies to properly interpret the content and understand the hierarchy of information on the page. Therefore, please ensure that you only use one `h1` element on your HTML page to adhere to best practices and accessibility guidelines :+1:.

Reply:
For sure! thank you for the such progressive review!